src/app/parcels/ParcelsPage.tsx

```diff
@@ -358,6 +359,27 @@ const ParcelsPage: React.FC<{}> = () => {
         }
     }, [parcelId]);
 
+    useEffect(() => {
+        const fetchAndSetClientIdForSelectedParcel = async (): Promise<void> => {
+            const { data, error } = await supabase
+                .from("parcels")
+                .select("client_id")
+                .eq("primary_key", parcelId)
+                .single();
+
+            if (error) {
+                void logError("...");
```

Comment:
Sorry I wasn't clear before, let's log something useful in `logError` (which might be called something else now if i remember correctly)

Maybe some error message, and also pass the `error` itself?

Reply:
Apologies, I copied your suggestion as a placeholder and forgot to replace the ellipsis. Added error text as well as the message.